src/main/config/config_master.h

```diff
@@ -99,6 +98,8 @@ typedef struct master_t {
     uint8_t blackbox_device;
 #endif
 
+    int16_t beeper_off_flags;
```

Comment:
15 out of 16 bits are already used - it would be probably better to use 32 bits from start or `typedef` beeper_off_flags type so that it is easier to change in (near) future.